elastic/elastic-agent-shipper
# Add an Elasticsearch docker configuration for testing

## What does this PR do?

This is the Elasticsearch docker configuration I'm using for the shipper integration tests, part of #176. I'm submitting it standalone to simplify review and to make sure the essential components are merged before the impending holiday. It is based on container configurations in the `beats` and `elastic-package` repositories.

- [ ] ~~I have added an entry in `CHANGELOG.md` or `CHANGELOG-developer.md`.~~

## How to test this PR locally

From `testing/environments/elasticsearch`, start the container:

```sh
ELASTICSEARCH_IMAGE_REF="docker.elastic.co/elasticsearch/elasticsearch:8.6.0-SNAPSHOT" docker-compose up -d
```

Verify that it's reachable:

```sh
curl -u admin:testing -f https://localhost:9200 --insecure
```
